cmd/loadTests.go

```diff
@@ -486,9 +501,9 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 			}
 
 			ComponentDetectionQueryName := fmt.Sprintf("%s-cdq", username)
-			cdq, err := framework.AsKubeDeveloper.HasController.CreateComponentDetectionQuery(ComponentDetectionQueryName, usernamespace, QuarkusDevfileSource, "", "", "", false)
+			cdq, err := framework.AsKubeDeveloper.HasController.CreateComponentDetectionQueryWithTimeout(ComponentDetectionQueryName, usernamespace, QuarkusDevfileSource, "", "", "", false, 60*time.Minute)
 			if err != nil {
-				logError(6, fmt.Sprintf("Unable to create ComponentDetectionQuery %s: %v", cdq.Name, err))
```

Comment:
`cdq` might be `nil` and that would cause an invalid memory access panic. So replacing it with already existing value.